r-build.md

```diff
@@ -0,0 +1,177 @@
+---
+highlighter: none
+layout: default
+title: Building R in HTC
+---
+
+# Overview
+
+**This guide will provide instructions for compiling base R from source code and is intended 
+for compiling versions of R that are currently not provided via the CHTC Squid Web Proxy**
+
+Before compiling R, please take a moment to review our [R Jobs](http://chtc.cs.wisc.edu/r-jobs.shtml) guide to see what versions of R are currently available on the HTC.
+
+**To best understand the below information, users 
+should already have an understanding of how to:**
+  * navigate within directories 
+  * create/copy/move/delete files and directories 
+  * run your intended program 
+  * use HTCondor to submit jobs
+
+This guide details the steps needed to: 
+
+ 1. [Download R source code](#1-download-r-source-code) 
+ 2. [Submit an interactive job to a build server](#2-submit-an-interactive-job-to-a-build-server) 
+ 3. [Compile base R from source code](#3-compile-base-r-from-source-code) 
+ 4. [Install additional R packages](#4-install-packages) 
+ 5. [Create a portable copy of your R installation that can be brought along with your jobs](#5-create-a-portable-copy-of-your-r-installation)
+ 6. [Use your custom R installation in your jobs](#6-use-your-custom-r-installation-in-your-jobs) 
+
+# Compile R
+
+## 1. Download R Source Code
+
+Before running any commands on CHTC, use a browser to get the source code 
+for your desired version of R from [CRAN](https://cran.r-project.org) 
+Under "Source Code for all Platforms", find the R-#.#.#.tar.gz file for your 
+desired version of R and download it to your computer before transferring a copy 
+to your `/home` directory on the submit server. 
```

Comment:
Seems simpler to have them `wget` it (or whatever) directly to the submit machine (i.e., `wget https://cran.r-project.org/src/base/R-4/R-4.0.1.tar.gz`). Any particular reason to have them download to their computer first?

Reply:
I was just sticking with the directions from our previous R build guide (since I was in a hurry). We recently made changes in our OSG documentation to remove wget from wrapper scripts in favor of using transfer_input_files, so it may be that we want to due the same here?

Reply:
@ChristinaLK, what do you think the guidance should be going forward?

Reply:
I think this is fine as is.